src/sidebar/helpers/test/mention-suggestions-test.js

```diff
@@ -27,6 +34,18 @@ describe('combineUsersForMentions', () => {
         displayName: 'Cecelia Davenport',
       },
     ];
+    const usersWhoWereMentioned = [
+      {
+        userid: 'acct:johndoe@example.com',
+        username: 'johndoe',
+        displayName: 'John Doe',
+      },
+      {
+        userid: 'acct:cecilia92@example.com',
+        username: 'cecilia92',
+        displayName: 'Cecelia Davenport',
```

Comment:
Not that it matters at all, but I notice the spelling  of "cecilia" differs between the username and display name.

Reply:
Oops 😅